src/bptl/conf/base.py

```diff
@@ -370,3 +373,24 @@
 
 # project application settings
 MAX_TASKS = 10
+
+
+# api settings
+REST_FRAMEWORK = {
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+        "rest_framework.renderers.BrowsableAPIRenderer",
```

Comment:
this one should only be included in the dev settings

Reply:
"rest_framework.renderers.BrowsableAPIRenderer" is placed into dev settings